docs/Contribute.md

```diff
@@ -0,0 +1,126 @@
+	call myTask  
+	}  
+```  
+* To use syntax highlighting, include the language after the first three backticks (\`\`\`).  
+_Example:_   
+	\`\`\`json  
+		\{  
+		    "MyWorkflow.MyTask.VariableTwo": "Variable2"  
+		\}  
+	\`\`\`	
+_produces this block_  
+```json  
+	{
+	    "MyWorkflow.MyTask.VariableTwo": "Variable2"
+	}
+```	
+
+**REST API:**
+
+1. Edit the [`cromwell.yaml`](https://github.com/broadinstitute/cromwell/blob/develop/engine/src/main/resources/swagger/cromwell.yaml) to make any changes to the REST API content.  
+2. Regenerate the REST API markdown file by running `sbt generateRestApiDocs` from the main Cromwell directory.
+3. Commit both the changes to the `cromwell.yaml` and the (re)generated [RESTAPI.md](https://github.com/broadinstitute/cromwell/blob/develop/docs/api/RESTAPI.md).
+4. Once your branch is merged to the [`develop` branch](https://github.com/broadinstitute/cromwell/tree/develop), you will see your changes on the [REST API page](api/RESTAPI/).
+
+**Left-side menu:**
+
+To add or remove items from the menu, edit [mkdocs.yml](https://github.com/broadinstitute/cromwell/blob/develop/mkdocs.yml) in Cromwell.
+
+**Images**
+
+* Relative: `![](ImgName.png)`
+	* _Example:_ `![](jamie_the_cromwell_pig.png)` _produces this image_  
+	![](jamie_the_cromwell_pig.png)  
+* Absolute: `![](URLofImg.png)`
+	* _Example:_ `![](https://www.broadinstitute.org/sites/all/themes/custom/at_broad/logo.png)` _produces this image_  
+	![](https://www.broadinstitute.org/sites/all/themes/custom/at_broad/logo.png)
+
+### FAQs
+
+**_Why isn't my documentation showing up?_**
+
+* **Have you confirmed that your PR is merged?**  
+If not, [go merge it](https://github.com/broadinstitute/cromwell/pulls). This will trigger an automatic build.  
+
+* **Has the build finished?**   
+[Check build status here](https://readthedocs.org/projects/cromwell/builds/).
+
+* **Did you add the file(s) to the YAML file?**  
+If not, [add it here](https://github.com/broadinstitute/cromwell/blob/develop/mkdocs.yml).
+
+**_How do I add to the REST API documentation?_**
+
+* **Don't forget to regenerate**  
+After you edit the [`cromwell.yaml`](https://github.com/broadinstitute/cromwell/blob/develop/engine/src/main/resources/swagger/cromwell.yaml), run `sbt generateRestApiDocs` and commit all changes.  
+_Hint:_ Once you have regenerated the docs correctly, the hidden timestamp at the top of the [`RESTAPI.md` file](https://raw.githubusercontent.com/broadinstitute/cromwell/develop/docs/api/RESTAPI.md) will show the current time.
+
+**_How do I generate a Markdown document of the Swagger YAML?_**
+
+A library named [swagger2markup](https://github.com/Swagger2Markup/swagger2markup) will render a Swagger YAML into various formats, including Markdown. Then generated Markdown is further customized by a custom SBT task.
+
+**_Generating and Hosting ScalaDoc_**
+
+Essentially run `sbt doc` then commit the generated code into the `gh-pages` branch on this repository
```

Comment:
Essentially? 🙁

Reply:
I copy-pasta'd, I'll admit. 

What is the ScalaDoc for? Do we use it? Does anyone use it?

Reply:
We should nix it. If we ever want to resurrect we can, however since our github pages was removed a couple of years ago I know that no one is using it right now :)

Reply:
I would **guess** that it lets you import cromwell or cromwell-wdl and get access to API docs. I know at least one team is DSDE does this, maybe we can ask whether they ever use the scaladocs?